public/about.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Test-Guru</title>
+    <base target="_blank">
+  </head>
+  <body>
+    <h2>Test-Guru</h2>
+    <p>Описание:<br>
+      Создание тестов и управление ими <br>
+      Аутентификация<br>
+      Базовое разграничение прав доступа<br>
+      Выполнение тестов<br>
+      Публикация результатов
+    </p>
+    <a href="http://www.thinknetica.com">Наши курсы </a>
```

Comment:
```suggestion
    <a href="http://www.thinknetica.com" target="_blank">Наши курсы </a>
```
я бы скорее на уровне ссылки сделал, но ок, пусть будет в "хеад"